server/controllers/visitation.controllers.js

```diff
@@ -1,23 +1,44 @@
-import scheduleVisitation from '../services/visitation.service';
-// import { sendMail } from '../services/mailer.service';
-// import EMAIL_CONTENT from '../../mailer';
+import dotenv from 'dotenv';
+import { scheduleVisitation, getAllVisitations } from '../services/visitation.service';
+import EMAIL_CONTENT from '../../mailer';
+import { sendMail } from '../services/mailer.service';
 import httpStatus from '../helpers/httpStatus';
 
+dotenv.config();
+
+const { BALLERS_EMAIL } = process.env;
+
 const VisitationController = {
   book(req, res, next) {
     const booking = req.locals;
     const { user } = req;
     scheduleVisitation({ ...booking, userId: user._id })
       .then((schedule) => {
-        // sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, user, {
-        //   link: `http://ballers.ng/activate?token=${token}`,
-        // });
+        const contentBottom = `
+          <strong> Name: </strong>: ${schedule.visitorName}\n, 
```

Comment:
`\n` will be displayed as a string as this is an HTML email.
```suggestion
          <strong> Name: </strong>: ${schedule.visitorName}<br />, 
```